src/api/order/order.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { Order, OrderSelect } from './model';
+import { OrderArgs, OrderCreateInput } from './dto';
+import { PrismaService } from 'src/shared/datasource/prisma/prisma.service';
+
+@Injectable()
+export class OrderService {
+  constructor(private readonly prismaService: PrismaService) {}
+
+  public async findOne(
+    { where }: OrderArgs,
+    { select }: OrderSelect,
+  ): Promise<Order> {
+    return this.prismaService.order.findUnique({
```

Comment:
Same like in category, let's take the user info from the auth token instead of taking it from the query input, actually we can remove the user relationships from the input args because it will always be taken from the user token.